### ISSUE TITLE
jitstress test failure: JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh throws an exception

### ISSUE THREAD

**Issue by sbomer** [dotnet/runtime#64587]
Some more context in https://github.com/dotnet/runtime/issues/60154 which is about a timeout in the same testcase, but got conflated with failures similar to those below.

@VincentBu in https://github.com/dotnet/runtime/issues/60154#issuecomment-1020704084

Failed again in: [runtime-coreclr jitstress 20220123.1](https://dev.azure.com/dnceng/public/_build/results?buildId=1568028&view=ms.vss-test-web.build-test-results-tab&runId=44082314&resultId=110117&paneView=debug)

Failed test:
```
CoreCLR OSX x64 Checked zapdisable @ OSX.1014.Amd64.Open

- JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh
- JIT/HardwareIntrinsics/General/Vector256/Vector256_r/Vector256_r.sh
```
**Error message:**
```
Fatal error. System.Runtime.InteropServices.SEHException (0x80004005): External component has thrown an exception.
at JIT.HardwareIntrinsics.General.VectorBooleanBinaryOpTest__GreaterThanAllDouble.RunBasicScenario_UnsafeRead()
at JIT.HardwareIntrinsics.General.Program.GreaterThanAllDouble()
at JIT.HardwareIntrinsics.General.Program.Main(System.String[])
task_for_pid(70905) FAILED 5 (os/kern) failure
/private/tmp/helix/working/A90B08E6/w/AE9F097C/e/JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh: line 417: 70905 Abort trap: 6           (core dumped) $LAUNCHER $ExePath "${CLRTestExecutionArguments[@]}"

Return code:      1
Raw output file:      /tmp/helix/working/A90B08E6/w/AE9F097C/uploads/Reports/JIT.HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.output.txt
Raw output:
BEGIN EXECUTION
/tmp/helix/working/A90B08E6/p/corerun -p System.Reflection.Metadata.MetadataUpdater.IsSupported=false Vector256_ro.dll ''
Beginning test case Abs.Byte at 1/24/2022 2:17:52 AM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 1/24/2022 2:17:53 AM
Beginning test case Abs.Double at 1/24/2022 2:17:53 AM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 1/24/2022 2:17:53 AM
Beginning test case Abs.Int16 at 1/24/2022 2:17:53 AM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 1/24/2022 2:17:53 AM
Beginning test case Abs.Int32 at 1/24/2022 2:17:53 AM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 1/24/2022 2:17:53 AM
Beginning test case Abs.Int64 at 1/24/2022 2:17:53 AM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 1/24/2022 2:17:53 AM
Beginning test case Abs.SByte at 1/24/2022 2:17:53 AM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: Run


Stack trace
   at JIT_HardwareIntrinsics._General_Vector256_Vector256_ro_Vector256_ro_._General_Vector256_Vector256_ro_Vector256_ro_sh()
```

**Comment by JulieLeeMSFT** [comment]
@echesakovMSFT PTAL.

**Comment by echesakovMSFT** [comment]
I don't see this issue in runtime-coreclr jitstress pipeline: 
- [#20220207.1](https://dev.azure.com/dnceng/public/_build/results?buildId=1598837&view=results)
- [#20220206.1](https://dev.azure.com/dnceng/public/_build/results?buildId=1596091&view=results)
- [#20220205.1](https://dev.azure.com/dnceng/public/_build/results?buildId=1595244&view=results)

The macOS all failed but due to enormous waiting time for the Helix machine.

**Comment by echesakovMSFT** [comment]
The backlog on macOS queue is being resolved (see #65000). 
Nothing actionable on this item, closing.
